engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/views/product-details.html

```diff
@@ -46,6 +46,11 @@ <h2 translate>{{ product.name }}</h2>
                  tooltip-animation="false"
                  tooltip-append-to-body="true">
               </i>
+              <i class="fa fa-question-circle" ng-switch-when="last_affected_repo_on_filter"
+                 uib-tooltip="{{ 'You cannot remove this product because it has repositories that are last affected repository on content view filters' | translate }}"
```

Comment:
```suggestion
                 uib-tooltip="{{ 'You cannot remove this product because it has repositories that are the last affected repository on content view filters' | translate }}"
```

Reply:
Updated!